site/app/templates/admin/admin_gradeable/AdminGradeableBase.twig

```diff
@@ -123,7 +123,7 @@
         updateNavItemsStatus();
         $('[name="type"]').change(updateNavItemsStatus);
         $('[name="ta_grading"]').change(updateNavItemsStatus);
-
+        $('[name="regrade_allowed"]').change(updateNavItemsStatus);
```

Comment:
updateNavItemStatuses is only to update the display of the navigation controls.  Since regrade requests doesn't change which nav tabs the user sees, it doesn't need to call this on change.